lib/standard/text/abstract_text.nit

```diff
@@ -231,14 +231,99 @@ abstract class Text
 	#     assert "abcd".has_suffix("bcd")	     ==  true
 	fun has_suffix(suffix: String): Bool do return has_substring(suffix, length - suffix.length)
 
-	# If `self` contains only digits, return the corresponding integer
+	# Removes the underscores in `self`
+	#
+	#     assert "__init__".remove_underscores == "init"
+	fun remove_underscores: Text do
```

Comment:
This is strangely specific, and longer than the usual `replace("_")`. You should either make it private or change it to `remove(char: Char)`. Also the synopsis is off, it doesn't remove the underscore in self but returns a copy wtihout the underscores...